index.html

```diff
@@ -6,11 +6,73 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
+      <body class="body">
 
-<body>
-    <!-- Add your markup here -->
-</body>
-
-</html>
\ No newline at end of file
+          <!--HEADER-->
+          <header class="header">
```

Comment:
Well done on separating the sections into header, main, and footer using proper semantic HTML. I would do exactly the same!